### ISSUE TITLE
Change the OSC message

### ISSUE THREAD

**Issue by 19Marcus87** [Andymann/ofxOscMidi#3]
Hi!
Nice to meet you. I tried your app and it works perfectly; but i would like to change the osc message to match that one that the target APP need. Could it be possible? Thanks!

**Comment by Nytra** [comment]
Have just encountered this. Was expecting it to be "/noteOn/.../..." but then looked at the code and it actually needs to be "/NoteOn/.../..."

**Comment by andymann** [comment]
Hey, could you go into detail a little more?

**Comment by 19Marcus87** [comment]
Yeah, sure! I would like to use ofxOSCmidi to send osc messages on local network to control softwares such as Arena Resolume . Arena Is pre configured to listen to OSC messages that match the address :"/composition/*" . I was asking if it's possibile  to customize the the osc messages Path to map midi signals that i send to ofxOSCmidi. Thanks for your interest.

**Comment by benjaminauer** [comment]
there are solutions by using a middleman, like OSC Router https://github.com/ETCLabs/OSCRouter

**Comment by andymann** [comment]
Good morning everyone in here. I am very sorry to not having answered on this. It just slipped through. I did not intentionally neglect any of you. @19Marcus87 I will have another look at it. @benjaminauer thanks for jumping in with a helpful answer

**Comment by andymann** [comment]
Hey @19Marcus87 , I just had a quick look into the code. Let's be honest, I will probably not find the time to apply any changes within the next months. However, here are the places in the code where you can add/ alter the response toincoming and outgoing messages:

// manipulate repsonse to INcoming messages
https://github.com/Andymann/ofxOscMidi/blob/3e76dd8bdc5dc9d063f1cd86f7f034e072dba66b/src/ofApp.cpp#L575

https://github.com/Andymann/ofxOscMidi/blob/3e76dd8bdc5dc9d063f1cd86f7f034e072dba66b/src/ofApp.cpp#L579

https://github.com/Andymann/ofxOscMidi/blob/3e76dd8bdc5dc9d063f1cd86f7f034e072dba66b/src/ofApp.cpp#L582

// manipulate OUTgoing messages
https://github.com/Andymann/ofxOscMidi/blob/3e76dd8bdc5dc9d063f1cd86f7f034e072dba66b/src/ofApp.cpp#L344

https://github.com/Andymann/ofxOscMidi/blob/3e76dd8bdc5dc9d063f1cd86f7f034e072dba66b/src/ofApp.cpp#L382

The following link points to another repo of mine where you'll find some info on how to compile the app yourself so you do not have to wait for me to get this done:
https://github.com/Andymann/ofxMidiClockAbletonLink
